docs/src/index.md

```diff
@@ -10,6 +10,21 @@ end
 
 *StatsBase.jl* is a Julia package that provides basic support for statistics. Particularly, it implements a variety of statistics-related functions, such as scalar statistics, high-order moment computation, counting, ranking, covariances, sampling, and empirical density estimation.
 
+## Installation
+
+To install StatsBase through the Julia REPL, you can type `] add StatsBase` or:
+
+```julia
+using Pkg
+Pkg.add("StatsBase")
+```
+To load the package, use the command:
+
+```
+using StatsBase
+```
+
+## Available methods
```

Comment:
"Methods" is ambiguous in this context: I guess you mean "statistical methods", but it's used in other places to refer to Julia methods, and yet this also covers types. Maybe this?
```suggestion
## Available Features
```